### PR TITLE
605 send push notifications in batches from back end

### PR DESCRIPTION
**Pull request by rd3fa**
We believe this is code complete but are not sure about the best way to test.

### CODE REVIEW

File: Sensus.sln [hunk 1]
```diff
@@ -30,27 +30,27 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Sensus.iOS.Shared", "Sensus
 EndProject
```
Comment (MatthewGerber) [inline Sensus.sln]: @clmckinley Can you revert the .sln file?

Comment (rd3fa) [reply Sensus.sln]: @MatthewGerber  this should be done now. Thanks!

**Comment by MatthewGerber** [comment]
@rd3fa This will also require changes to the backend script that processes the push notification requests:

https://github.com/predictive-technology-laboratory/sensus/blob/develop/Scripts/ConfigureAWS/send-push-notifications.sh

**Comment by rd3fa** [comment]
@MatthewGerber  For the changes requested when I click “see review” it bumps me up to the .sln file revert. This is done. Was there something else that needs to be reviewed?

For the bash script, is there any way that we can get some sample source files to run the script against?

**Pull request closed without merging**